packages/field-plugin/package.json

```diff
@@ -43,6 +43,9 @@
     "test": "vitest",
     "build": "unbuild"
   },
+  "dependencies": {
+    "pure-parse": "^1.0.1"
```

Comment:
Question here: We are using `valibot` for all plugins right now. I know @johannes-lindgren, you have been proposing to use yours for a long time now, but since the current plugin ecosystem uses `valibot` in all projects, shall we stick with that one in this case?

I'm just saying because adding it here means that from now on, the rest of the plugins will use this one when creating new projects. However, the naming convention is different for the validators, so we will have different conventions for different plugins, and it is a change in paradigm every time you work on fixes/testing.

Maybe also a question for @jeremycolin & @demetriusfeijoo  🙏

Reply:
I'd say let's stick to what we use for now so let's go with valibot